app/helpers/application_helper.rb

```diff
@@ -75,4 +76,29 @@ def render_flash(flash)
   rescue StandardError
     flash.inspect if Rails.env.development?
   end
+
+  # Generates a set of social share buttons based on the specified options.
+  def render_social_share_buttons(opts = {})
+    sites = opts.fetch(:allow_sites, [])
+    valid_sites, invalid_sites = SocialShareButtonHelper.filter_allowed_sites(sites)
+
+    # Log invalid sites
+    invalid_sites.each do |invalid_site|
+      Rails.logger.error("Invalid site or icon not configured: #{invalid_site}")
+    end
+
+    tag.div(
+      :class => "social-share-button d-flex gap-1 align-items-end flex-wrap"
+    ) do
+      valid_sites.map do |site|
+        link_to SocialShareButtonHelper.generate_share_url(site, opts), {
+          :rel => ["nofollow", opts[:rel]].compact,
+          :class => "ssb-icon",
+          :title => "Share on #{site.capitalize}"
+        } do
+          image_tag(SocialShareButtonHelper.icon_path(site), :alt => site.capitalize, :size => 28)
```

Comment:
`:alt => site.capitalize`: "email" is not a proper site name, you'll have to translate it instead of just capitalizing. 

You'll have something like:
https://github.com/openstreetmap/openstreetmap-website/blob/1b288d40c7fff3d3be8d82d4e96c9008778f4dba/config/locales/en.yml#L2597-L2614